README.md

```diff
@@ -392,20 +392,19 @@ How does async-flow work? It does the following:
   3. It requests that all `:events` mentioned in `flow` rules should be "forwarded" to
      this event handler, after they have been handled by their normal handlers.
      So, if the event `:abc` was part of `flow` spec, then after `[:abc 1]` was handled by its normal handler
-     there would be an additional `(dispatch [:async/flow  [:abc 1]])` which would be handled the coordinator
+     there would be an additional `(dispatch [:id [:abc 1]])` which would be handled the coordinator
```

Comment:
Here I've replaced `:async/flow` with `:id`. According to code this is what really happens.